msocasecharge.yaml

```diff
@@ -0,0 +1,176 @@
+entityDefinitions:
+
+  person:
+    fqn: "nc.SubjectIdentification"
+    entitySetName: "MSOInmates"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "PCP"
+    name: "person"
+ 
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "MSOJailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SYSID"
+    name: "jailstay"
+
+  charge:
+    fqn: "ol.offense"
+    entitySetName: "MSOCharge"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "CHARGE_PK"
+      ol.eventcount:
+        type: "ol.eventcount"
+        column: "CHARGE_ORDER"
+      ol.type:
+        type: "ol.type"
+        column: "CHARGE_TYPE_ID"
+    name: "charge"
+
+  chargeevent:
+    fqn: "ol.chargeevent"
+    entitySetName: "MSOChargeevent"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "CHARGE_PK"
+      ol.code:
+        type: "ol.code"
+        column: "OFFENSE_CODE"
+      ol.description:
+        type: "ol.description"
+        column: "OFFENSE_DESCRIPTION"
+      ol.datetimestart:
+        type: "ol.datetimestart"
+        column: "OFFENSE_DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S", "yyyy-MM-dd HH:mm:ss.SS", "yyyy-MM-dd HH:mm:ss.SSS"]
+          timezone: "America/New_York"
+      date.completeddatetime:
```

Comment:
this sounds like the date written into the system and may have nothing to do with the charge. I think we need to ignore it or find a more appropriate association for it.

Reply:
deleted it for now